services/src/error.rs

```diff
@@ -56,6 +56,9 @@ pub enum Error {
     Proj {
         source: proj::ProjError,
     },
+    Trace {
+        source: opentelemetry::trace::TraceError,
+    },
```

Comment:
Can't we wrap that into an Opentelemetry error?

Reply:
This IS the opentelemetry error.

Reply:
It is a trace error, right?
Aren't there other errors with respect to our spectacular remote logging?

Reply:
I don't know if it is `spectacular` but remote logging is called "tracing" and the `trace` module of opentelemetry is the "API for tracing applications and libraries." and thus the ` TraceError` should be exactly what we need.